extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/runtime/HttpServerConfig.java

```diff
@@ -58,23 +59,23 @@ public class HttpServerConfig {
      *
      * @asciidoclet
      */
-    @ConfigItem(defaultValue = "false")
-    public boolean suppress4xxErrors;
+    @WithDefault("false")
+    boolean suppress4xxErrors();
 
     /**
      * Maximum number of unique URI tag values allowed. After the max number of
      * tag values is reached, metrics with additional tag values are denied by
      * filter.
      */
-    @ConfigItem(defaultValue = "100")
-    public int maxUriTags;
+    @WithDefault("100")
+    OptionalInt maxUriTags();
 
-    public void mergeDeprecatedConfig(VertxConfig config) {
-        if (!ignorePatterns.isPresent()) {
-            ignorePatterns = config.ignorePatterns;
-        }
-        if (!matchPatterns.isPresent()) {
-            matchPatterns = config.matchPatterns;
-        }
+    default void mergeDeprecatedConfig(VertxConfig config) {
+        //        if (!ignorePatterns().isPresent()) {
+        //            ignorePatterns = config.ignorePatterns;
+        //        }
+        //        if (!matchPatterns().isPresent()) {
+        //            matchPatterns = config.matchPatterns;
+        //        }
     }
```

Comment:
Hey @mcruzdev , what's the status of this one? If this is the only problem remaining, I can probably sort it out and we could get it merged. Let me know!

Reply:
Hi @gsmet, feel free to change it